docs/source/usage/preparation.rst

```diff
@@ -34,22 +39,22 @@ To deploy this framework you will need the following established:
 
       See :doc:`deployment` section for more details.
 
-    4. **A CodeCommit or Github repository:**
+    5. **A CodeCommit or Github repository:**
 
       This is where your configuration files will be stored in addition to the buildspec and deploy scripts.
       If you parameterize all your sensitive data, there shouldn't be any risk of sensitive data disclosure.
 
       .. note:: The env: variables that must be filled out, these can essentially be whatever you want them to be provided they fall in line with the naming conventions the cloudformation and serverless scripts expect.
 
 
-    5. **GitHub Framework Repo:**
+    6. **GitHub Framework Repo:**
 
       This repo contains all the templates you'll need to build out the pipeline. 
       The cloudformation template will automatically build your CI/CD pipeline which will deploy the serverless application. 
       You should not need to make any changes to it unless you plan to modify it for your own purposes, which is entirely OK with us!
 
 
-    6. **Pick a Cloudformation template:**
+    7. **Pick a Cloudformation template:**
```

Comment:
With this https://github.com/mozilla/mozdef-event-framework/pull/23/commits/5173cf49fb2046d22237a2388cc714024cc1696d in #23 we changed it so that a codecommit repo is not created by the multi source template anymore (needs to be created manually). We should delete the `note` section below.

Reply:
removed